File: atcoder/src/B_PowerSocket.cs
// https://atcoder.jp/contests/abc139/tasks/abc139_b
#nullable disable

class B_PowerSocket
{
	static void Main()
	{
		Console.WriteLine(Solution());
	}

	static String Solution()
	{
		int[] ab = Console.ReadLine().Split().Select(int.Parse).ToArray();
		int tab = ab[0];
		int need = ab[1];
		if(need == 1) return "0";
		if(need == 2) return "1";
		if(tab == 2) return "-1";
		double gap = need - tab;
		double up = tab - 1;
		return (Math.Ceiling(gap / up) + 1).ToString();
	}
}

File: clang/B_PowerSocket.cpp
// https://atcoder.jp/contests/abc139/tasks/abc139_b

#include <stdio.h>

int main(void)
{
  int a, b, guchi = 1, result = 0;
  scanf("%d %d", &a, &b);
  while (guchi < b) 
  { 
    guchi += (a-1);
    result++;
  }
  printf("%d", result);
  return 0;
}
